source/_patterns/01-molecules/13-multioffer-teaser/02-multioffer-teaser-offer.mustache

```diff
@@ -5,7 +5,7 @@
     {{> atoms-multioffer-teaser-hotel-content}}
     {{> atoms-multioffer-teaser-info}}
   </div>
-  {{> molecules-button-linked-price }}
+  {{> molecules-button-linked-price-arrow }}
```

Comment:
Please leave a comment in the Main Ticket, that you needed to change this in the Multioffer-Teaser. Otherwise noone will test it. And we want all changes to be tested.

Reply:
No worries. HTML-wise nothing changes in the multi-offer teaser. I just changed the name of the molecule, which has the same content as before.

Reply:
Nevertheless we often had Problems at other places, that were not visible in the Ticket. So if we change anything in a part of the Styleguide, that is not clear by reading the ACs of the Ticket, we should mention it somewhere at least. No matter, how small the change is. And yes, I already saw before, that it's actually the same, but what, if there is a typo, a copy-paste-error or something else? Then we maybe have again a situation like no stars on prod for more than a week.